pkg/cmd/roachtest/roachstress.sh

```diff
@@ -135,26 +135,28 @@ cr="${abase}/cockroach${local}"
 # the same SHA multiple times and artifacts shouldn't mix.
 a="${abase}/$(date '+%H%M%S')"
 
+[ -z "${short}" ] && artifact_suffix= || artifact_suffix=-short
+
 if [ ! -f "${cr}" ] || [ "${force_build}" = "y" ]; then
   if [ -z "${local}" ]; then
-    ./build/builder.sh mkrelease amd64-linux-gnu "build${short}"
-    cp "cockroach${short}-linux-2.6.32-gnu-amd64" "${cr}"
+    ./dev build ${short} --cross=linux
```

Comment:
Maybe

```
targ="cockroach"
if [ -n "${short}" ]; then
  targ="cockroach-${short}"
fi
./dev build "${targ}"
```

I see that `./dev build` and `./dev build short` also work but I wonder how canonical that is. After all, at some point, wouldn't you expect `./dev build` to "build all the things"?

```

Reply:
./dev build's default behavior without any target arguments is to build cockroach. That shouldn't change going forward.

Reply:
It is now `./dev build short` or `./dev build cockroach short` with cockroach being optional on either variants. I can make it verbose for clarity as we do multiple build calls here.

Reply:
Oh disregard what I said. I got it all wrong.